api/workers/autoscale.go

```diff
@@ -72,10 +75,7 @@ func autoscaleRack() {
 
 	err = models.Provider().SystemSave(*system)
 	if err != nil {
-		log.Log("fn=system.Save err=%q", err)
+		log.Error(err)
 		return
 	}
-
-	// log for humans
-	fmt.Printf("who=\"convox/monitor\" what=\"autoscaled instance count to %d\" why=\"a service wants %s processes behind a load balancer\"\n", system.Count)
```

Comment:
These have been helpful for debugging... The goal is to have a simple rack logs filter we can document so users can understand some internal changes that are executed and why